pgxn/neon/communicator_new.c

```diff
@@ -177,6 +178,32 @@ assign_request_id(void)
 	return result;
 }
 
+/*
+ * Returns the absolute path to the given path.
+*/
+static inline char * get_absolute_path(const char *path)
+{
+	char *abs_path = bounce_buf();
+	char cwd[PATH_MAX];
+	int size = 0;
+
+	if (path[0] == '/') {
+		strncpy(abs_path, path, strlen(path));
```

Comment:
```suggestion
		strlcpy(abs_path, path, strlen(path));
```
Just as a matter of principle. Even when used safely, strncpy() tends to light up all kinds of static analysis tools these days.